main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -346,7 +346,7 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
   // Lists of paths to the source files, flix packages and .jar files used
   private var sourcePaths: List[Path] = List.empty
   private var flixPackagePaths: List[Path] = List.empty
-  private var mavenPackagePaths: List[Path] = List.empty
+  private var jarPackagePaths: List[Path] = List.empty
```

Comment:
I think I would still prefer to have two separate lists, so one knows from where each is coming.

Reply:
Okay. I am just not sure how to distinguish the two "types" of jar files in folderMode() - any ideas?